src/components/Note.jsx

```diff
@@ -0,0 +1,98 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {faTrash, faPencil} from '@fortawesome/free-solid-svg-icons';
+import Swal from 'sweetalert2'
+import { useState } from 'react';
+import Button from './Button';
+function Note(props){
+    let {nt} = props;
+    let [isEditing, setIsEditing] = useState(false);
+    let [newValue, setNewValue] = useState(nt.content)
+    let date, time;
+    function extractDateAndTime(dateStr){
+        const objDate = new Date(dateStr);
+        date = objDate.toISOString().split('T')[0];
+        time = objDate.getHours() + ":" + objDate.getMinutes();
+    }
+    extractDateAndTime(nt.createdAt)
+
+    function handleDelete(id){
+        Swal.fire({
+            title: 'Cation!',
+            text: 'Are you sure you want delete this Note ?',
+            icon: 'warning',
+            footer: "Note: you cant undo this operation",
+            showDenyButton:true,
+            denyButtonText:"No",
+            confirmButtonText:"yes sure",
+            confirmButtonColor:"#14d1d1"
+          }).then((data)=>{
+            if(data.isConfirmed == true){
+                fetch(`http://localhost:3100/notes/${id}`,{
+                    method:"delete"
+                   })
+                   .then((res)=>res.json())
+                   .then(
+                       Swal.fire({
+                           title: 'Success',
+                           text: 'Your Note was deleted successfully',
+                           icon: 'success',
+                           confirmButtonText:"OK",
+                           confirmButtonColor:"#14d1d1"
+                         })
+                   )
+                   .then(
+                       props.setYourNotes(
+                           props.yourNotes.filter(el=>
+                               el._id !== id
+                           )
+                       )
+                   )
+            }
+          })
+    }
```

Comment:
use axios instead of fetch, 
try catch and async await instead of ,then
place the confirmations message in its component
follow the SRP in this handler